tasks/src/Types.ts

```diff
@@ -19,6 +19,7 @@ export type TaskDemoEntry = {
 	filename: string;
 	type: "img";
 } | {
+	label : string,
```

Comment:
This is why the CI is failing in moon-landing @merveenoyan - this adds a mandatory `label` key for when `type: "tabular"`, but there's no such `label` in `table-question-answering`, `tabular-classification` and `tabular-regression`.

So either add a label in those cases or make it optional here (`label?: string;`) - you probably know better than me what's best :)